WalletWasabi.Fluent/Views/Wallets/Home/History/TransactionDetailsView.axaml

```diff
@@ -36,8 +36,8 @@
       <c:PreviewItem Icon="{StaticResource status_regular}"
                      Text="Status">
         <Panel>
-          <TextBox Classes="selectableTextBlock" TextWrapping="NoWrap" Text="Confirmed" IsVisible="{Binding IsConfirmed}" />
-          <TextBox Classes="selectableTextBlock" TextWrapping="NoWrap" Text="Pending" IsVisible="{Binding !IsConfirmed}" />
+          <TextBlock TextWrapping="NoWrap" Text="Confirmed" IsVisible="{Binding IsConfirmed}" />
+          <TextBlock TextWrapping="NoWrap" Text="Pending" IsVisible="{Binding !IsConfirmed}" />
```

Comment:
`TextWrapping` could be removed. That was a temp fix for the selectable stuff because it showed `Pendin` instead of `Pending.`